src/stash/src/upgrade/v19_to_v20.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::{StashError, Transaction, TypedCollection};
+
+pub mod objects_v20 {
+    include!(concat!(env!("OUT_DIR"), "/objects_v20.rs"));
+}
+
+const MZ_SYSTEM_ROLE_ID: objects_v20::RoleId = objects_v20::RoleId {
+    value: Some(objects_v20::role_id::Value::System(1)),
+};
+const ACL_MODE_CREATE_CLUSTER: u64 = 1 << 29;
+const ACL_MODE_CREATE_DB: u64 = 1 << 30;
+const ACL_MODE_CREATE_ROLE: u64 = 1 << 31;
+const MZ_SYSTEM_ACL_MODE: objects_v20::AclMode = objects_v20::AclMode {
+    bitflags: ACL_MODE_CREATE_CLUSTER | ACL_MODE_CREATE_DB | ACL_MODE_CREATE_ROLE,
+};
```

Comment:
Isn't it possible to use the constants from src/repr/src/adt/mz_acl_item.rs here? It seems that stash depends on repr if I'm not mistaken.

Reply:
The upgrades aren't supposed to import structs from external crates. If the struct definitions change the upgrades may no longer be valid or make sense.